Formula/riak.rb

```diff
@@ -44,7 +44,7 @@ class Riak < Formula
 
   resource "yokozuna" do
     url "https://github.com/basho/yokozuna.git",
-        :revision => "b53d999529626301c36fa3efa22b2b0165217556"
+        :tag      => "riak_kv-2.9.0"
```

Comment:
Again, can we use the `:revision` that this tag points to? Also, add a comment with the tag to clarify.